Убийство эскадрильи и псиопа дионей

Lezzerd opened the pull request:
### Описание изменений

Невероятно обрезал дионам пилота и псионика.
Почему?
Потому что дионы не умеют играть в майкрософт флай симулятор. 
А ещё они качались в биологическое возвышение а не в псионическое.

### Почему и что этот ПР улучшит

Уберегаем торч от профессиональных пилотов с пятисекундными задержками между маневрами на овермапе.
А ещё спасаем  бедных лороведов от споров касательно псионической силы дион.
### Авторство

ЯЙЦО
### Чейнджлог

Дионы теряют пилота и псионика.

#### Требуемые изменения на википедии проекта

Ничего????

Lezzerd committed:
Убийство эскадрильи и псиопа дионей

Убрал пилота и псионика. Почему? Потому что дионы не качались в псионическое возвышение.

Exapsters commented:
А у нас могут быть дионеи следователями?

Exapsters commented:
Если да, то это проблема, ведь у нас есть поддолжности на следователя с псионикой.

Exapsters commented:
И на священника, к слову, тоже.

Lezzerd committed:
Продолжение террора

Отобрал священника и следователя из-за того, что им кто-то выдал пси-подпрофы.
Уроды

Lezzerd commented:
Убрал ещё и их, ладно.

Lezzerd committed:
Отбираем ещё и у машинок, чтобы не было скучно

даааа
